src/main/scala/de/mukis/sbt/slick/SlickCodeGenH2.scala

```diff
@@ -15,6 +16,15 @@ object SlickCodeGenH2 extends AutoPlugin {
     slickJDBCDriver := "org.h2.Driver",
     slickUrl := { db =>
       s"jdbc:h2:mem${db.map(":" + _) getOrElse ""}"
+    },
+    slickArguments := { database =>
+      val driver = slickDriver.value
+      val jdbc = slickJDBCDriver.value
+      val url = slickUrl.value
+      val outputDir = (sourceManaged in Compile).value.getPath
+      val rootPackage = slickPackage.value
+
+      Array(driver, jdbc, url(database), outputDir, rootPackage)
```

Comment:
This order has to be correct in all other plugins as well, which is error prone if changes need to be made.